### ISSUE TITLE
connecting bug with yeh fina 

### ISSUE THREAD

**Issue by Gue3bara** [alif-type/mada#24]
<img width="205" alt="screenshot 2017-07-26 01 43 51" src="https://user-images.githubusercontent.com/12478169/28598446-f087b1fa-71a3-11e7-9a23-97a30fbf8550.png">
<img width="222" alt="screenshot 2017-07-26 01 46 03" src="https://user-images.githubusercontent.com/12478169/28598495-3b2ef1fa-71a4-11e7-946d-840551aa9d25.png">
<img width="203" alt="screenshot 2017-07-26 01 46 12" src="https://user-images.githubusercontent.com/12478169/28598496-3b5efc10-71a4-11e7-9b74-b41fede7ce45.png">
basically this is the only bug i came across while testing mada

**Comment by khaledhosny** [comment]
Not a bug as we already discussed in https://github.com/alif-type/mada/issues/10.
